tiled/catalog/adapter.py

```diff
@@ -372,6 +377,11 @@ async def asset_by_id(self, asset_id):
         return Asset.from_orm(asset)
 
     def structure(self):
+        if self.structure_family == StructureFamily.container:
+            # Give no inlined contents.
+            return ContainerStructure(contents=None, count=None)
+        if self.structure_family == StructureFamily.composite:
+            return CompositeStructure(contents=None, count=None)
```

Comment:
Not sure about this. I need to dig in to remind myself when this is called, but off the top of my head I'd think that `composite` nodes should _always_ include inlined contents (because it will never be huge).